CHANGELOG.md

```diff
@@ -4,5 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.2] - 2017-03-14
+Corrected API docs.
+
+### Fixed
+- Issue where CodeClimate would consider underscore-prefixed private methods to
+be breaking the naming convention.
```

Comment:
There aren't, it's true. But I did this for consistency, so that we have the same config "base" across all repos. I understand it's extremely unlikely for there to be prefixed methods in this repo. But in case there ever are, we won't have to wonder why we have CC complain about them. And if we decide to copy a config from somewhere and choose this repo, we won't get an outdated one.